packages/project-access/test/project/cap.test.ts

```diff
@@ -24,6 +24,17 @@ import type { Logger } from '@sap-ux/logger';
 
 jest.mock('child_process');
 const childProcessMock = jest.mocked(childProcess, { shallow: true });
+const jestMockEnv = {
+    'for': jest.fn().mockImplementation(() => ({
+        'for': jest.fn().mockImplementation(() => ({
+            'for': jest.fn().mockImplementation(() => ({
+                folders: { app: 'MY_APP', db: 'MY_DB', srv: 'MY_SRV' }
+            })),
+            folders: { app: 'MY_APP', db: 'MY_DB', srv: 'MY_SRV' }
+        })),
+        folders: { app: 'MY_APP', db: 'MY_DB', srv: 'MY_SRV' }
+    }))
+};
```

Comment:
Hi @devinea, 
sorry, missed this in my first review. This 3 levels of `for` recursion is required because this is the max amount is it called? 

Here is a proposal that works independent of the number of calls:
```suggestion
const jestEnvForMock = jest.fn().mockImplementation(() => ({
    'for': jestEnvForMock,
    folders: {
        app: 'MY_APP',
        db: 'MY_DB',
        srv: 'MY_SRV'
    }
}));
const jestMockEnv = {
    'for': jestEnvForMock
};
```